react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -119,6 +126,13 @@ export default class JitsiStreamBackgroundEffect {
                 this._inputVideoElement.width,
                 this._inputVideoElement.height
             );
+        }
+        if (this._options.virtualBackground.backgroundType === 'desktop-share') {
+            this._outputCanvasCtx.drawImage(
```

Comment:
I think we'll need to play a bit with coordinates here right? What happens when a small window is shared? Smaller than the available canvas size that is.

Reply:
Added for now the _inputVideoElement coordinates like on 'image' case.

Reply:
Found a solution. Done.

Reply:
Or maybe we need to do something like on simple desktop share?